### PR TITLE
Add probe for vpn SSL stats

### PR DESCRIPTION
**Pull request by IMSdevsecu**
Hi,

We are interested in listing connected user stats to SSL VPN for example in Grafana dashboards.
This code add support for probing SSL VPN stats.

### CODE REVIEW

File: pkg/probe/vpn_ssl_stats_test.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/testutil"
+)
+
+func TestVPNSslStats(t *testing.T) {
+	c := newFakeClient()
+	c.prepare("api/v2/monitor/vpn/ssl/stats", "testdata/vpn-stats.jsonnet")
+	r := prometheus.NewPedanticRegistry()
+	if !testProbe(probeVPNSslStats, c, r) {
+		t.Errorf("probeSystemStatus() returned non-success")
```
Comment (bluecmd) [inline pkg/probe/vpn_ssl_stats_test.go:16]:
```suggestion
		t.Errorf("probeVPNSslStats() returned non-success")
```

**Review by bluecmd — CHANGES_REQUESTED** [review]
LGTM with minor notes

File: pkg/probe/vpn_ssl_stats.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	Results VPNResults `json:"results"`
+	VDOM    string     `json:"vdom"`
+	Version string     `json:"version"`
+}
+
+func probeVPNSslStats(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.Metric, bool) {
+	var (
+		vpnCurUsr = prometheus.NewDesc(
+			"fortigate_vpn_ssl_current_users",
+			"Number of current VPN users",
```
[the remark this answers is not in the document]
Comment (IMSdevsecu) [reply pkg/probe/vpn_ssl_stats.go:30]: Sure, this will avoid some confusion

**Comment by bluecmd** [comment]
I am OK with setting the value to 0 or a small value (<100) by default. Either way works for me.

Can you rebase (not merge) on the latest? It seems there are conflicts and the diffs are all messed up sadly.

File: README.md [hunk 3]
```diff
@@ -59,9 +59,13 @@ Per-VDOM:
...
    * `fortigate_interface_transmit_errors_total`
    * `fortigate_interface_receive_errors_total`
- * _VPN/Ssl_
+ * _VPN/Ssl/Connections_
    * `fortigate_vpn_connections`
    * `fortigate_vpn_users`
+ * _VPN/Ssl/Stats_
+   * `fortigate_vpn_ssl_current_users`
+   * `fortigate_vpn_ssl_current_tunnels`
+   * `fortigate_vpn_ssl_current_connections`
```
Comment (bluecmd) [inline README.md:68]: The `current` part is implied, and the other VPN metric does not have it, so we should drop it.

```suggestion
   * `fortigate_vpn_ssl_users`
   * `fortigate_vpn_ssl_tunnels`
   * `fortigate_vpn_ssl_connections`
```

Comment (IMSdevsecu) [reply README.md:68]: Ok, this was just to differenciate the 2 sections of the API call
```
  {
    "http_method":"GET",
    "results":{
      "conserve_mode":false,
      "max":{
        "users":7,
        "tunnels":6,
        "connections":12
      },
      "current":{
        "users":3,
        "tunnels":2,
        "connections":2
      }
    },
```
Ok for dropping it

**Review by bluecmd — APPROVED** [review]

**Pull request merged**